test/lib/study.js

```diff
@@ -55,6 +55,110 @@ module.exports = function(client, config) {
             .waitForPaginationComplete();
     };
 
+    me.view.visits = {
+        /**
+         * Fill out a study's visit form.
+         *
+         * This method works for both 'create' and 'edit' visit forms. The only
+         * thing that changes is the `form`'s `id`.
+         *
+         * @param  {Object} options Contains `data` and `mode` properties.
+         *   `data` contains input data for the form. `mode` holds the input
+         *   type of the form, either `add` for adding a new visit or `update`
+         *   for updating a visit. Example:
+         *
+         *       {
+         *           data: {
+         *               label: 'Test Label 2',
+         *               timeFromBaseline: 2,
+         *               timeUnit: 'Month'
+         *           },
+         *           mode: 'update'
+         *       }
+         *
+         * @return {Object}         Instance of `client`
```

Comment:
no rework req'd.  we recently decided to lowercase primatives, FYI. not sure if that made it back int our docs.  sexy doc blocks!